js/donations-visualiser.js

```diff
@@ -887,6 +887,12 @@ function processData(data) {
     filterData();
 }
 
+
+function toTrainCase(str) {
+    if (!str || typeof str !== 'string') return str;
+    return str.toLowerCase().replace(/ /g, '-');
```

Comment:
no harm in handling tabs, too! I didn't consider double spaces, but prob wouldn't rewrite `hello&nbsp;&nbsp;tim` to `hello-tim`.. maybe ambiguous.. am i missing something there? I've got one space mapping to one dash globally, right?